### DISCUSSION TITLE
plays/password-generator

### DISCUSSION THREAD

**Discussion by giscus** [reactplay/react-play#152] [Play Feedback]
# plays/password-generator

Its a simple password generator built in react using what user can generate password and customize their requirements in choosing characters and number while generating a medium or strong level password.

https://www.reactplay.io/plays/password-generator

**Comment by atapas** [comment]
This is a great project. Thanks for adding.

**Comment by shrilakshmishastry** [comment]
What a coincident you created play on the occasion of world password day 😃

**Reply by atapas** [reply]
So true.

**Comment by Asterisk-z** [comment]
I have an idea for a design for this password generator can i go ahead and implement on your codebase

**Reply by atapas** [reply]
Could you please share your design first? It will be great to see that @Asterisk-z. I think the easiest thing will be open up a discussion here: https://github.com/atapas/react-play/discussions

Thanks a lot for showing interest.

**Reply by atapas** [reply]
It is being discussed here: https://github.com/atapas/react-play/discussions/214
